Sources/StreamChatUI/ChatMessageList/Attachments/AttachmentViewCatalog.swift

```diff
@@ -14,8 +14,7 @@ open class AttachmentViewCatalog {
         message: ChatMessage,
         components: Components
     ) -> AttachmentViewInjector.Type? {
-        // Do not show attachments for deleted messages
-        guard !message.isDeleted else { return nil }
+        if message.isDeleted { return nil }
```

Comment:
nit: in general, I wouldn't include changes that are not directly connected to the topic. In this case it's fine but i it would complicate review process for large PRs.. 

Regarding the change, when you see `guard` you expect to see early return OR throwing an error while `ifs` do not trigger `early return` expectation and are good for checks inside a function body.

Reply:
- guard not deleted else -> exit
- if deleted -> exit

second form looks much more readable